libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -127,6 +127,7 @@ include AP_Notify/AP_Notify.h
 singleton AP_Notify alias notify
 singleton AP_Notify method play_tune void string
 singleton AP_Notify method handle_rgb void uint8_t 0 UINT8_MAX uint8_t 0 UINT8_MAX uint8_t 0 UINT8_MAX uint8_t 0 UINT8_MAX
+singleton AP_Notify method handle_scr_disp void uint8_t 0 5 string
```

Comment:
This is a bad name for the method from a script side. Something like `set_display_text()` maybe?